addons/mail/static/src/messaging/entity/chatter/chatter.js

```diff
@@ -106,10 +106,7 @@ function ChatterFactory({ Entity }) {
          * @returns {boolean}
          */
         _computeIsDisabled() {
-            if (!this.threadId) {
-                return true;
-            }
-            return this.isDisabled;
+            return !this.threadId;
```

Comment:
Actually, it feels strange to have `threadId` stored in the composer instead of just `thread`, but this is out of scope.